libs/Microsoft.MixedReality.WebRTC.Native/src/peer_connection.cpp

```diff
@@ -1188,4 +1271,9 @@ ErrorOr<RefPtr<PeerConnection>> PeerConnection::create(
   return RefPtr<PeerConnection>(peer);
 }
 
+void PeerConnection::SetFrameHeightRoundMode(FrameHeightRoundMode value) {
+  InitHololensH264Workaround();
```

Comment:
See other comment, there is supposed to be another call in PeerConnection::create. Calling it here initializes the static bool in InitHololensH264Workaround so that a following call to PeerConnection::create won't reset the value.